pkgs/by-name/py/pyflyby/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "pyflyby";
+  version = "1.9.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "deshaw";
+    repo = "pyflyby";
+    rev = version;
+    hash = "sha256-QkoFr9tFtZ+ZEWlxe9csrzoYFl9/V2l4hKYfUWsXUdc=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [
+    setuptools
+    wheel
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    six
+    toml
+    isort
+    black
```

Comment:
Do we need black as a runtime dep?

Reply:
It's an odd runtime dependency but build fails without it
```
       > Checking runtime dependencies for pyflyby-1.9.6-py3-none-any.whl
       >   - black not installed
```